modules/explorer/info.go

```diff
@@ -38,6 +42,32 @@ func (e *Explorer) BlockFacts(height types.BlockHeight) (modules.BlockFacts, boo
 	return bf.BlockFacts, true
 }
 
+// LatestBlockFacts returns a set of statistics about the blockchain as they appeared
+// at the latest block height in the explorer's consensus set.
+func (e *Explorer) LatestBlockFacts() modules.BlockFacts {
+	var bf blockFacts
+	err := e.db.View(func(tx *bolt.Tx) error {
+		var height types.BlockHeight
+		err := encoding.Unmarshal(tx.Bucket(bucketInternal).Get(internalBlockHeight), &height)
+		if err != nil {
+			return err
+		}
+		block, exists := e.cs.BlockAtHeight(height)
```

Comment:
so... present module design (which is bad, and I wish to fix soon) means that it's a deadlock risk to call an exported consensus function while holding an explorer lock, which we should be holding. I believe though that the explorer has an internal way of looking up the block at a given height.